Detectors/MUON/MCH/Raw/Decoder/include/MCHRawDecoder/DecodedDataHandlers.h

```diff
@@ -28,6 +28,12 @@ using SampaChannelHandler = std::function<void(DsElecId dsId,
                                                uint8_t channel,
                                                SampaCluster)>;
 
+/// A SampaChannSampaHeartBeatHandler is a function that takes a chip index and
+/// a bunch crossing counter value found in a HeartBeat packet
```

Comment:
I'm getting a bit confused here :  why do we need the chip here ? Does it mean the HeartBeat can actually be at different times for the 2 halves of a given dual sampa ?

Reply:
Normally not, but each chip sends it's own HeartBeat packet in response to an HB trigger, so we can keep track of the bunch crossing counter of each individual chip and eventually check that there are no mismatches.